ysoserial/Generators/WindowsPrincipalGenerator.cs

```diff
@@ -113,7 +115,7 @@ public override object Generate(string formatter, InputArgs inputArgs)
             }
             else if (formatter.ToLower().Equals("datacontractserializer"))
             {
-                string payload = $@"<root type=""System.Security.Principal.WindowsPrincipal, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"">
+                string payload = $@"<root type=""System.Security.Principal.WindowsPrincipal"">
```

Comment:
Please do not shorten it by hand - the minifier should take care of this (we need to keep it useful for researchers when reading it)